packit/config/package_config.py

```diff
@@ -194,7 +194,26 @@ def load_packit_yaml(
         return safe_load(raw_text) or {}
     except YAMLError as ex:
         logger.error(f"Cannot load package config {config_file_path}.")
-        raise PackitConfigException(f"Cannot load package config: {ex!r}.")
+        if hasattr(ex, "problem_mark"):
+            if ex.context is not None:  # type: ignore
+                logger.error(
+                    "  parser says\n"
+                    + str(ex.problem_mark)
+                    + "\n  "
+                    + str(ex.problem)  # type: ignore
+                    + " "
+                    + str(ex.context)  # type: ignore
+                )
+            else:
+                logger.error(
+                    "  parser says\n"
+                    + str(ex.problem_mark)
+                    + "\n  "
+                    + str(ex.problem)  # type: ignore
+                )
```

Comment:
I haven't tried because there's no reproducer but I naively assume this should be equivalent:

```suggestion
            msg = f"  parser says\n{str(ex.problem_mark)}\n  {str(ex.problem)}"
            if ex.context is not None:  # type: ignore
                msg += f" {str(ex.context)}"
            logger.error(msg)
```

Reply:
The reproducer is easy. Just make any typo in yaml file, e.g., put a random string at the start of an indented line. And call `packit validate-config`.
I am going to update this as you suggested.